node/src/components/linear_chain_sync/blocks.rs

```diff
@@ -0,0 +1,176 @@
+use tracing::warn;
```

Comment:
I think the entire `linear_chain_sync` component has been removed on the `feat-fast-sync` branch.